packages/rancher-grafana/generated-changes/exclude/ci/with-dashboard-json-values.yaml

```diff
@@ -0,0 +1,53 @@
+dashboards:
+  my-provider:
+    my-awesome-dashboard:
+      # An empty but valid dashboard
+      json: |
+        {
+          "__inputs": [],
+          "__requires": [
+            {
+              "type": "grafana",
+              "id": "grafana",
+              "name": "Grafana",
+              "version": "6.3.5"
```

Comment:
JW: The version in the chart for grafana is 6.6.4 - does this version have anything to do that the actual version of grafana?

Reply:
Yes, this is the version of Grafana that the dashboards were created for.

But there's no backwards compatibility with Grafana, so these dashboards can still be installed in 6.6.4 even if they are meant for 6.3.5. So I don't think this field has an actual impact on the dashboard